web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Prisma File
+---
+
+TODO: create a diagram showing how the Wasp file takes entities from the Prisma file
+
+TODO: talk about how Wasp understands the Prisma file
```

Comment:
I think we will have some repetition to what we said in entities.md. But we can see, some repetition is probably fine, and some stuff we can just link to.